<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ihm_XBEE</class>
 <widget class="QMainWindow" name="ihm_XBEE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_analyzer">
       <attribute name="title">
        <string>Analyser</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QCheckBox" name="trace_active">
          <property name="text">
           <string>Active trace</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTextEdit" name="xbee_trace">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="clear_trace">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_generator">
       <attribute name="title">
        <string>Generator</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Données utiles à envoyer.&lt;/p&gt;&lt;p&gt;Format : Decimal / hexa ou mixte, séparation par des espaces.&lt;/p&gt;&lt;p&gt; Ex: 100 0xab 12 245 0xef 0x9F 45&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Data to encode and send</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>87</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Destination address</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>127</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="send_data">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QLineEdit" name="line_generator"/>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="destination_address">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="prefix">
           <string>0x</string>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>65535</number>
          </property>
          <property name="displayIntegerBase">
           <number>16</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_config">
       <attribute name="title">
        <string>Config</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="9" column="0">
         <widget class="QCheckBox" name="auto_config_startup">
          <property name="text">
           <string>Auto configure at startup</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>CHANNEL</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="channel">
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>COORDINATOR</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="coordinator">
          <property name="maxLength">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>COORDINATOR_OPTION</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="coordinator_option">
          <property name="maxLength">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>PANID</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="panid">
          <property name="maxLength">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>KEY</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="key">
          <property name="maxLength">
           <number>32</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="id">
          <property name="maxLength">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>SECURITY</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="security">
          <property name="maxLength">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>API_MODE</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="api_mode">
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QPushButton" name="send_config">
          <property name="text">
           <string>Send config</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
